src/core_functions/aggregate/distributive/arg_min_max.cpp

```diff
@@ -324,16 +327,12 @@ AggregateFunction GetArgMinMaxFunctionInternal(const LogicalType &by_type, const
 template <class OP, class ARG_TYPE>
 AggregateFunction GetArgMinMaxFunctionBy(const LogicalType &by_type, const LogicalType &type) {
 	switch (by_type.InternalType()) {
-	case PhysicalType::INT8:
-		return GetArgMinMaxFunctionInternal<OP, ARG_TYPE, int8_t>(by_type, type);
-	case PhysicalType::INT16:
-		return GetArgMinMaxFunctionInternal<OP, ARG_TYPE, int16_t>(by_type, type);
 	case PhysicalType::INT32:
 		return GetArgMinMaxFunctionInternal<OP, ARG_TYPE, int32_t>(by_type, type);
 	case PhysicalType::INT64:
 		return GetArgMinMaxFunctionInternal<OP, ARG_TYPE, int64_t>(by_type, type);
-	case PhysicalType::FLOAT:
-		return GetArgMinMaxFunctionInternal<OP, ARG_TYPE, float>(by_type, type);
```

Comment:
I would avoid removing INT16 (in particular) since it's used by GetDecimalArgMinMaxFunction. Also INT8 and FLOAT probably are fine staying.

Reply:
I removed them to reduce the code footprint back to what it was before I made the fix.

Reply:
The impact on runtime memory is minimal because we only store one value. There is some casting overhead but that is the same as for non-`DECIMAL` types.

Reply:
Right, forgot it was in the connected PR. All cool then.